virtualenv.py

```diff
@@ -1395,6 +1395,20 @@ def install_python(home_dir, lib_dir, inc_dir, bin_dir, site_packages, clear, sy
                 elif os.path.exists(python_dll_d_dest):
                     logger.info("Removed %s as the source does not exist", python_dll_d_dest)
                     os.unlink(python_dll_d_dest)
+
+            if not IS_PYPY:
+                # considering that on windows, python import libraries are located in
+                # the  "<root>/libs" directory, the following code will look for and
+                # copy "pythonXY.lib".
+                pythonlib_name = "python{}{}.lib".format(sys.version_info[0], sys.version_info[1])
+                pythonlib = os.path.join(os.path.dirname(sys.executable), "libs", pythonlib_name)
+                pythonlib_dest_dir = os.path.join(lib_dir, "..", "libs")
```

Comment:
Shouldn't this take the lib_dir value from https://github.com/pypa/virtualenv/blob/508919a789dcb43469bdfbdcbf9d98546b4c076c/virtualenv.py#L1095?

Reply:
The `.lib` file indeed lives in the `.libs` folder.

![image](https://user-images.githubusercontent.com/219043/51206318-c55fc780-18d5-11e9-8471-17a4817a91b5.png)